docs/en/manuals/live-update.md

```diff
@@ -127,218 +61,46 @@ When bundling, any excluded resource will be left out of the application bundle.
 
-If `resource.store_manifest` succeeds, the new manifest is now located in local storage. The next time the engine starts this manifest will be used instead of the one bundled with the game.
+## Content verification
 
-### Caveats
-There are a few gotchas that might be good to know if you plan to use this feature to store a new manifest with Live update.
+One of the major features of the our live update system, is that you can use now use many content archives, potentially from many different Defold versions.
 
-* It is only possible to add or modify resources referenced by collection proxies that are tagged as `Exclude` in the new manifest. No changes should be made to already bundled resources or resources not in excluded collection proxies. For example, doing changes in a script that is referenced by a bundled collection will cause the resource system to look for that resource in the bundled data archive. But since the shipped game bundle has not changed (only the manifest has) the changed script will not be found and consequently cannot be loaded.
+The `liveupdate.add_mount()` default behavior, is to add an engine version check when attaching a mount.
+This means that both the game base archive and liveupdate archive(s) need to be created at the same time with the same engine version. This will invalidate any previously downloaded archives by the client, forcing them to redownload the content.
 
-* Even though this feature allows you to quickly push bug fixes or new features to a live game without doing a full app store release, it should be used with care. Pushing out a new manifest should involve the same processes as when doing a full release with everything that that entails (testing, QA, etc.).
+This behavior can be turned off with an options flag.
+When turned off, the content verification responsibility lies entirely with the developer, to guarantuee that each liveupdate archive will work with the running engine.
 
-## Setting up Amazon Web Service
+We recommend storing some metadata for each mount, so that _directly upon startup_, the developer can decide if the mount/archive should be removed.
+One way to do so is to add an extra file to the zip archive after the game has been bundled. For instance by inserting a `metadata.json` with any relevant information that the game requires. Then, at startup, the game can retrieve with `sys.load_resource("/metadata.json")`. _Note that you will need a unique name for each mount's custom data, or the mounts will give you the file with the topmost priority_
 
-To use the Defold Live update feature together with Amazon services you need an Amazon Web Services account. If you don't already have an account you can create one here https://aws.amazon.com/.
+Failure to do so, you may end up in a situation where the content is not compatible with the engine at all, forcing it to quit.
```

Comment:
Will it quit or will the collection proxy load fail? I can't remember. I suppose it depends?

Reply:
If it crashes, it will quit.
I don't want to say that I think?